packages/portal/src/components/generic/PaginationNavInput.vue

```diff
@@ -81,6 +120,22 @@
       maxResults: {
         type: Number,
         default: null
+      },
+      buttonIcons: {
```

Comment:
When would we not want icons?
Looks like the intention was to use this prop to allow passing a class in so that we can have the circled or plain arrows.
At the moment this isn't used.

Reply:
Well spotted. I've refactored it to `buttonsIconClass` to allow passing in the desired icon class.